### ISSUE TITLE
temporary containers

### ISSUE THREAD

**Issue by slynobody** [ghacksuserjs/ghacks-user.js#804]
current version does not allow to install '[temporary containers](https://addons.mozilla.org/en-US/firefox/addon/temporary-containers/?src=search)' (a really good privacy-extension to completely isolate web-sites)

reproduce:
open a new ff-account 
include user.js
try to install aforementioned ff-extension
result: grayed out extension; tab-close when trying to access extension-preferences

**Comment by Thorin-Oakenpants** [comment]
1394448 UA spoof: breaks AMO 💡 you can still install extensions

    drag the + Add to Firefox button to the urlbar
    right click the + Add to Firefox button and open in a new tab
        may fail with a message "Firefox prevented this site from asking you to install software on your computer" in which case just refresh the new tab

**Comment by crssi** [comment]
Add to override:
``` js
/* 4503  */ user_pref("privacy.resistFingerprinting.block_mozAddonManager", false); // enable AMO to work as intended, 2662 must be default
```

😉 
cheers

**Comment by Thorin-Oakenpants** [comment]
`privacy.resistFingerprinting.block_mozAddonManager` is not the problem

OP says the user.js doesn't allow installing the extension? Then OP says the extension is greyed out? Does he mean the ` + to Firefox ` button? Then I can understand what "tab-close when trying to access extension-preferences" - WTH does that even mean.

Anyway, in a brand new FF69 profile, I added the current user.js and restarted. Went to AMO, installed Multi-Account Containers, then installed Temporary Containers, and then went to the extension's options and was able to navigate and change all the settings

Closing as invalid: unless OP can be more precise
